modules/system/assets/js/framework.js

```diff
@@ -68,6 +68,11 @@ if (window.jQuery.request !== undefined) {
             requestHeaders['X-OCTOBER-REQUEST-FLASH'] = 1
         }
 
+        var csrfToken = getCSRFToken()
```

Comment:
@daftspunk as above, this should probably be `getXSRFToken` to make it clear it is separate to any CSRF tokens passed from input.